<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotterWidjet</class>
 <widget class="QWidget" name="PlotterWidjet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCustomPlot" name="PlotWidjet_1" native="true"/>
   </item>
   <item>
    <widget class="QCustomPlot" name="PlotWidjet_2" native="true"/>
   </item>
   <item>
    <widget class="QCustomPlot" name="PlotWidjet_3" native="true"/>
   </item>
   <item>
    <widget class="QScrollBar" name="PlotHorizontalScrollBar">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
